### ISSUE TITLE
file:// URL warning needs to be updated

### ISSUE THREAD

**Issue by fippo** [webrtc/samples#496]
from https://webrtc.github.io/samples/:
```
Please note that all samples that use getUserMedia() must be run from a server.
Calling getUserMedia() from a file:// URL will result in a PERMISSION_DENIED
NavigatorUserMediaError.
```
This doesn't seem to be true any longer. In fact, this just does nothing these days (@marwahvikas?). In any case, that text is probably obsolete, also given the ancient error name. A better way would be to warn inside adapter.

**Comment by samdutton** [comment]
You're right – good spot. See PR #497.

**Comment by KaptenJansson** [comment]
Fixed in #497
